src/server.c

```diff
@@ -711,7 +711,6 @@ static void on_connection_bound(qd_server_t *server, pn_event_t *e) {
             pn_ssl_domain_t* plugin_ssl_domain = NULL;
             if (config->sasl_plugin_config.use_ssl) {
                 plugin_ssl_domain = pn_ssl_domain(PN_SSL_MODE_CLIENT);
-
```

Comment:
Does it make sense to patch this file simply to remove whitespace?

Reply:
Sorry about that. Fixed now ! Thanks